xmlcommand/BESXMLInterface.cc

```diff
@@ -95,24 +95,21 @@ void BESXMLInterface::build_data_request_plan()
         xmlSetGenericErrorFunc((void *) &parseerrors, BESXMLUtils::XMLErrorFunc);
 
         // XML_PARSE_NONET
-        doc = xmlReadMemory(d_xml_document.c_str(), (int)d_xml_document.size(), "" /* base URL */,
+        doc = xmlReadMemory(d_xml_document.c_str(), (int) d_xml_document.size(), "" /* base URL */,
                             nullptr /* encoding */, XML_PARSE_NONET /* xmlParserOption */);
 
         if (doc == nullptr) {
-            string err = "Problem parsing the request xml document:\n";
+            std::string err = "Problem parsing the request xml document:\n";
```

Comment:
Why add the std:: namesake here? it's omitted elsewhere in this file...

Reply:
Good question. Should be removed